partner-chains-cli/src/ogmios/mod.rs

```diff
@@ -1,17 +1,22 @@
 use anyhow::anyhow;
 use jsonrpsee::http_client::HttpClient;
-use ogmios_client::{query_ledger_state::QueryLedgerState, query_network::QueryNetwork};
+use ogmios_client::{
+	query_ledger_state::QueryLedgerState, query_network::QueryNetwork, types::OgmiosUtxo,
+};
 
 #[derive(Debug, Eq, PartialEq)]
 pub enum OgmiosRequest {
 	QueryLedgerStateEraSummaries,
 	QueryNetworkShelleyGenesis,
+	QueryUtxo { address: String },
 }
 
 #[derive(Debug)]
 pub enum OgmiosResponse {
 	QueryLedgerStateEraSummaries(Vec<EraSummary>),
 	QueryNetworkShelleyGenesis(ShelleyGenesisConfiguration),
+	QueryUtxo(Vec<OgmiosUtxo>),
+	Error(String),
```

Comment:
Because this PR is quite big already and I'd like to have it dispatched in v1.3 I'm not dealing better with Ogmios and Context.

As fun exercise I'll try to bind request and response with phantom types and all introduce Result<OgmiosResponse, ...> instead of plain OgmiosResponse.

Reply:
I think I'll just change how IOContext works. It will have incjected mocked implementations of Ogmios clients